pkg/remoteclusters/reconcile_config.go

```diff
@@ -47,6 +47,7 @@ func (c *Cluster) reconcileConfig(remoteConfig *istiov1beta1.RemoteIstio, istio
 	istioConfig.Spec.Citadel.NodeSelector = remoteConfig.Spec.Citadel.NodeSelector
 	istioConfig.Spec.Citadel.Affinity = remoteConfig.Spec.Citadel.Affinity
 	istioConfig.Spec.Citadel.Tolerations = remoteConfig.Spec.Citadel.Tolerations
+	istioConfig.Spec.Citadel.CASecretName = CASecretName
```

Comment:
Shouldn't this be:
```suggestion
	istioConfig.Spec.Citadel.CASecretName = remoteConfig.Spec.Citadel.CASecretName
```
?

Or what happens if a user defines a different secret name in the remoteIstio CR than the Istio CR? Do we allow that?

Reply:
I pushed a commit which implements custom CA on the remote side as well.

Reply:
Thank you, @waynz0r!